File: examples/python/others/get_inboxes.py
from utils import MaiAgentHelper

API_KEY = '<your-api-key>'

assert API_KEY != '<your-api-key>', 'Please set your API key'


def main():
    maiagent_helper = MaiAgentHelper(API_KEY)

    inbox_items = maiagent_helper.get_inbox_items()
    maiagent_helper.display_inbox_items(inbox_items)


if __name__ == '__main__':
    main()

File: examples/csharp/others/get_inboxes.cs
using System;
using System.Diagnostics;
using System.Threading.Tasks;
using Utils;

namespace MaiAgentExamples.Others
{
    public static class GetInboxes
    {
        public static string API_KEY = "<your-api-key>";

        public static async Task Main(string[] args)
        {
            Debug.Assert(API_KEY != "<your-api-key>", "Please set your API key");

            var maiagentHelper = new MaiAgentHelper(API_KEY);
            var inboxItems = await maiagentHelper.GetInboxItemsAsync();
            maiagentHelper.DisplayInboxItems(inboxItems);
        }
    }
}
